### ISSUE TITLE
:host { display: flex } shown in the HTML content?

### ISSUE THREAD

**Issue by atypicalprogrammer** [electron/electron#7309]
* Electron version: latest.
* Operating system: Win 7, SP1

For a user on the above system, this CSS is shown inside, as content.. Not sure why. 
I noticed this CSS rule is added to a webview. Below is the normal behavior: 

index.html : 

`    <webview id="server" src="about:blank" preload="./js/server-ipc.js"></webview>
`

Rendered:

```
webview
   shadowroot
      <style>:host { display: flex }</style>
      <object is="browserplugin" type="application/browser-plugin" id="browser-plugin-1" style="flex: 1 1 auto;"></object>
  
```
![iotatest](https://cloud.githubusercontent.com/assets/20187287/18754868/73e4b18e-80ea-11e6-951d-aba159f6d785.png)

**Comment by zeke** [comment]
It looks like that's being set here: https://github.com/electron/electron/blob/f5694daeb793c41dea4ecf2a689a19aa2a9ad74d/lib/renderer/web-view/web-view.js#L34

```js
shadowRoot.innerHTML = '<style>:host { display: flex; }</style>'
```

Perhaps Windows 7 doesn't like style tags that don't have a `type` attribute?

**Comment by baconface** [comment]
`type` is required for HTML4 documents. My guess is `<!DOCTYPE html>` isn't present at the start of the page. Without it Chrome would think it is processing an HTML4 document instead of an HTML5 document.

**Comment by atypicalprogrammer** [comment]
No, the doctype is specified:

```
<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <link rel="stylesheet" href="css/style.css" />
  </head>
  <body>
    <webview id="server" src="about:blank" preload="./js/server-ipc.js"></webview>
    <div id="status-bar">
      <div id="status-bar-left">
        <span id="status-bar-transactions"></span>
        <span id="status-bar-dot" style="display:none"> • </span>
        <span id="status-bar-cpu"></span>
      </div>
      <div id="status-bar-right"><span id="status-bar-giota"></span></div>
    </div>
    <div id="new-user">
      New user? Read the <strong>FAQ</strong> first!
    </div>
  </body>
  <script src="bower_components/tingle/dist/tingle.min.js"></script>
  <script type="text/javascript">
    require("./js/index.js")
  </script>
</html>
```

**Comment by nikola** [comment]
For HTML5 documents, `type` is optional and defaults to `text/css` if it's missing. So declaring `<!DOCTYPE html>` should be sufficient.

**Comment by zeke** [comment]
@atypicalprogrammer the doctype may be specified in _your_ code, but it is not set in the HTML code that is being injected into the `shadowRoot` (whatever that is!). See https://github.com/electron/electron/pull/7313

**Comment by zcbenz** [comment]
Fixed with #7313.
